src/components/TodoListItem.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import styles from "./TodoListItem.module.css";
+import check from "../assets/pig-svgrepo-com.svg";
+import PropTypes from "prop-types";
+
+
+const TodoListItem = ({ todo, onRemoveTodo }) => {
+  const handleOnRemove = () => {
+    onRemoveTodo(todo.id);
+  };
+
+  return (
+    <li className={styles.listItem}>
+       <span>
+          {todo.title}
+      </span>
+      <span>
+        <button 
+          type="button" 
+          onClick={handleOnRemove}
+        >
+          <img src={check} className={styles.checkLogo} alt="Check logo" />
+        </button>
+      </span>
+    </li>
+  );
+};
+
+TodoListItem.propTypes = {
+  todoLtodoist: PropTypes.string,
```

Comment:
This prop type doesn't match the one you're using. You have a `todo` prop and it's an object, not a string.